xml/docu_styleguide.outline.xml

```diff
@@ -307,6 +296,24 @@
      </para>
     </listitem>
    </varlistentry>
+    <varlistentry>
+     <term>Contributors</term>
+     <listitem>
+      <para>
+       Writing the manuals is mostly a collaborative effort. To credit also
+       external contributors, add an appendix to the guides, based on the
+       <guimenu>Contributors</guimenu> page for the respective GitHub
+       repository. List the contributors in alphabetical order. Include a
```

Comment:
We should decide whether listing should be alphabetical by given name/family name/GitHub display name.
cwickert suggest: `@github-name: John Doe` (sorted by GitHub user name)

Reply:
In fact my idea was: `John Doe (<link xlink:href="https://github.com/jdoe">@jdoe</link>)`.

Reply:
So the Chicago Manual of Style basically says: Use the name the author chooses. Most will have >Firstname Lastname>, others Initials ("T. S. Elliot") etc. If we just use the name they set in git, we leave the decision to them and only need to care about certain corner cases.

Reply:
I have adjusted this as discussed in our last meeting on 2021-03-12. Including a link to the GH Contributor's page saves us the hassle of deciding for full names or GH user names (and how to sort them alphabetically) and deduplicating contributor names.